### PR TITLE
Add initial jekyll files and first blog post

### PR DESCRIPTION
**Pull request by tkyi**
This PR:
- Using https://jekyllrb.com/ to create a blog
- Includes first blog post "Why Screwdriver"
- Can be accessed at http://blog.screwdriver.cd

Some issues:
- Not sure if images work yet

### CODE REVIEW

**Comment by okuryu** [comment]
There is better file names for the first post like this:

* `_posts/2016-08-25-welcome-to-screwdriver.markdown`
* `_posts/2016-08-25-why-screwdriver.markdown`

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "3.2.1"
+
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem "minima"
+
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
```
Comment (stjohnjohnson) [inline Gemfile]: Given this line, should we follow it?

**Comment by stjohnjohnson** [comment]
I like it! 👍

File: Gemfile (hunk 1 above)
Comment (tkyi) [inline Gemfile]: No idea, let me try it out

**Comment by Filbird** [comment]
Looks clean! 👍 

giphy-2

**Comment by stjohnjohnson** [comment]
lots of cats

**Pull request merged**